src/theme.js

```diff
@@ -450,17 +450,17 @@
     });
   }
 
-  function moveTopBarContainer(to) {
-    waitForElements(
-      ['.main-topBar-container', '.Root__main-view'],
-      ([topBarContainer, rootMainView]) => {
-        rootMainView.insertAdjacentElement(
-          to === 'up' ? 'beforebegin' : 'afterbegin',
-          topBarContainer
-        );
-      }
-    );
-  }
+//   function moveTopBarContainer(to) {
+//     waitForElements(
+//       ['.main-topBar-container', '.Root__main-view'],
+//       ([topBarContainer, rootMainView]) => {
+//         rootMainView.insertAdjacentElement(
+//           to === 'up' ? 'beforebegin' : 'afterbegin',
+//           topBarContainer
+//         );
+//       }
+//     );
+//   }
```

Comment:
Why not remove it entirely?

Reply:
yeah i will do it

Reply:
@kernelb00t thanks for helping us out!!

@sanoojes lemme knw once you are done with it!

Reply:
okee